src/Microsoft.DotNet.Build.CloudTestTasks/RunnerScripts/scriptrunner/scriptrunner.py

```diff
@@ -26,27 +26,23 @@ def _main(settings, optlist, args):
             xunitrunner
                 [--config config.json]
                 [--setting name=value]
-                --script
-                [--args arg1 arg2...]
+                --script=path
+                [args]
         """
-        optdict = dict(optlist)
         log.info("BuildTools Helix Script Runner v0.1 starting")
-        if '--args' in optdict:
-            script_arguments = optdict['--args']
-            log.info("Script Arguments:"+script_arguments)
-
-        script_to_execute = optdict['--script']
 
-        test_executor = HelixTestExecution(settings)
+        optdict = dict(optlist)
 
+        script_to_execute = optdict['--script']
         unpack_dir = fix_path(settings.workitem_payload_dir)
-
         execution_args = [os.path.join(unpack_dir, script_to_execute)] + args
 
+        test_executor = HelixTestExecution(settings)
+
         return_code = helix.proc.run_and_log_output(
-            execution_args,
-            cwd=unpack_dir,
-            env=None
+       	    execution_args,
+       	    cwd=unpack_dir,
+       	    env=None
         )
         event_client = helix.event.create_from_uri(settings.event_uri)
```

Comment:
When running in a repro environment, we don't want the machine sending telemetry at all and will have not provided a value for the event URI in settings (if things went correctly...).  All the event_client usages here should be conditional on the presence or absence of an event_uri.